include/cantera/thermo/CoverageDependentSurfPhase.h

```diff
@@ -0,0 +1,617 @@
+    /*!
+     * After setting coverages, covstateNum is incremented by 1.
+     */
+    virtual void setCoveragesNoNorm(const doublereal* theta);
+
+    //! Return the covstateNum
+    int statecovNumber() const {
+        return m_covstateNum;
+    }
+
+    //! Convert given eneregy to the quantity with its default unit of J/kmol.
+    double convertEnergy(double value, const std::string& src) const;
+
+    //! Convert given energy per temperature to the quantity with its default unit of J/kmol/K.
+    double convertEnergy_T(double value, const std::string& src) const;
+
+    // Functions calculating reference state thermodyanmic properties--------------
+
+    //! Return the nondimensionalized reference state enthalpy.
+    /*!
+     * Nondimensionalized reference state enthalpy is evaluated at T, P_{ref} and \theta_{ref}:
+     *
+     * \f[
+     *      h^{ref}_k / RT = h_k(T, P_{ref}, \theta_{ref}) / RT
+     * \f]
+     */
+    virtual void getEnthalpy_RT_ref(doublereal* hrt) const;
+    //! Return the nondimensionalized reference state entropy.
+    /*!
+     * Nondimensionalized reference state entropy is evaluated at T, P_{ref} and \theta_{ref}:
+     *
+     * \f[
+     *      s^{ref}_k / R = s_k(T, P_{ref}, \theta_\text{ref}) / R
+     * \f]
+     */
+    virtual void getEntropy_R_ref(doublereal* sr) const;
+    //! Return the nondimensionalized reference state cp.
+    /*!
+     * Nondimensionalized reference state cp is evaluated at T, P_{ref} and \theta_{ref}:
+     *
+     * \f[
+     *      cp^{ref}_k / R = cp_k(T, P_{ref}, \theta_\text{ref}) / R
+     * \f]
+     */
+    virtual void getCp_R_ref(doublereal* cpr) const;
+    //! Return the nondimensionalized reference state Gibbs free energy.
+    /*!
+     * \f[
+     *      g^{ref}_k / RT = h^{ref}_k /RT - s^{ref}_k /R
+     * \f]
+     */    
+    virtual void getGibbs_RT_ref(doublereal* grt) const;
+
+    // Functions calculating standard state thermodyanmic properties---------------   
+
+    //! Return the nondimensionalized standard state enthalpy.
+    /*!
+     * Nondimensionalized standard state enthalpy is evaluated at T, P and \theta:
+     *
+     * \f[
+     *      h^o_k / RT = (h^{ref}_k + h^{cov}_k(T, P, \theta)) / RT
+     *                   + \int_{T_{ref}}^{T} cp^{cov}_k(T, P, \theta) dT / RT
+     * \f]
+     */
+    virtual void getEnthalpy_RT(doublereal* hrt) const;
+    //! Return the nondimensionalized standard state entropy.
+    /*!
+     * Nondimensionalized standard state entropy is evaluated at T, P and \theta:
+     *
+     * \f[
+     *      s^o_k / R = (s^{ref}_k + s^{cov}_k(T, P, \theta)) / R - ln(1 / \theta_{ref})
+     *                   + \int_{T_{ref}}^{T} cp^{cov}_k(T, P, \theta) / T dT / R
+     * \f]
+     */
+    virtual void getEntropy_R(doublereal* sr) const;
+    //! Return the nondimensionalized standard state cp.
+    /*!
+     * Nondimensionalized standard state cp is evaluated at T, P and \theta:
```

Comment:
`\theta` needs to be in math mode -- check the generated documentation to be sure that this looks right here and elsewhere.